Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -134,6 +137,14 @@ final class PurchasesOrchestrator {
         }
 
         Task {
+            #if os(iOS)
```

Comment:
I can't remember what the latest was, does this include catalyst and visionOS?

Reply:
I believe visionOS is not included. Will add it 👍

Reply:
Actually, I'm not sure if catalyst is included, since it seems it's added only in some places, even for APIs that seem supported there (like `beginRefundRequest` APIs)... Will try to figure that out.